Resources/Prototypes/_Goobstation/Entities/Objects/Specific/Christmas/lights.yml

```
@@ -0,0 +1,343 @@
+# I couldn't figure out how randomsprite works so this mess was created!!!
+# Also pointlight masks were probably created by yakub or some shit to fuck with people holy god
+# Pointlight masks were coded in hell and sent up to us ong
+- type: entity
+  id: SpawnerChristmasLightsWhite
+  parent: MarkerBase
+  components:
+  - type: Sprite
```

yamllint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)